plugins/Live/Controller.php

```diff
@@ -228,11 +228,42 @@ public static function getAllProfileSummaryInstances()
         if (!$cache->contains($cacheId)) {
             $instances = [];
 
+            /**
+             * Triggered to add new live profile summaries.
+             *
+             * **Example**
+             *
+             *     public function addProfileSummary(&$profileSummaries)
+             *     {
+             *         $profileSummaries[] = new MyCustomProfileSummary();
+             *     }
+             *
+             * @param ProfileSummaryAbstract[] $profileSummaries An array of profile summaries
+             */
+            Piwik::postEvent('Live.addProfileSummaries', array(&$instances));
```

Comment:
Not sure if we should introduce an event for adding summaries, as this is done automatically by looking for classes extending the `ProfileSummary` class.

Reply:
It is basically the same as for Visitor Details, Widgets, Reports, ... this way, it allows plugins that define entities, such as custom dimensions, to add for example a profile summary for each entity / custom dimension. It is just a general thing that we do so far as not always it is possible to have it in the plugin structure.

Reply:
ah sure. Guess I haven't thought enough before writing the comment...